### ISSUE TITLE
Azure CLI with Bicep works differently

### ISSUE THREAD

**Issue by justinyoo** [devcontainers/features#336]
Hi, Team.

When I added the `azure-cli` feature to the devcontainer settings, I enabled the following option:

* `"installBicep": true`

Because it's a part of Azure CLI option, I was expecting that I would be able to use the following command:

```bash
az bicep version
```

However, it doesn't work. Rather it works in the following way:

```bash
bicep --version
```

It means that the Bicep CLI works as a standalone CLI instead of the part of Azure CLI. And I found the following codebase:

https://github.com/devcontainers/features/blob/7fa90110d762797cc0b1c2fe8fcc028c9b813d56/src/azure-cli/install.sh#L205-L208

Is there a reason that we use the standalone Bicep CLI, rather than Azure CLI integrated one?

**Comment by samruddhikhandale** [comment]
`bicep` is accessed with `bicep --version` and `az bicep version`. 

Repro 👇 

```jsonc
"features": {
    "ghcr.io/devcontainers/features/azure-cli:latest": {
      "installBicep": "true"
    }
  }
```

![image](https://user-images.githubusercontent.com/24955913/208179994-bbbc4da8-4c14-4e70-a783-7992917c2cc0.png)

@justinyoo  Are you passing any other option to this Feature? Could you help with repro steps and your dev config. If you add machine info details along with VS Code version, Extension details would be great.

**Comment by justinyoo** [comment]
I use this `devcontainer.json` file

https://github.com/devkimchi/power-platfform-connector-via-apim/blob/main/.devcontainer/devcontainer.json

The main difference from yours is that I use the tag of `:1` and yours is `:latest`.

**Comment by samruddhikhandale** [comment]
> Does :1 recognise the latest version?

Yes, we use semantic versioning. Hence, all points to the same artifact.

![image](https://user-images.githubusercontent.com/24955913/208476178-5965f514-603d-4673-b3b7-ca324b58ef06.png)

@justinyoo Your postcreatCommad is reinstalling bicep [here](https://github.com/devkimchi/power-platfform-connector-via-apim/blob/main/.devcontainer/post-create.sh#L32). I wonder if that's causing the problem? 

ℹ️ We have bumped the major version for `common-utils` to `2` with support for Debian/Ubuntu, RHEL/Fedora/Rocky/CentOS, and Alpine Linux! Alsim. Alos, we have renamed `uid` & `gid` option to fix a bug. You may want to switch to that?

https://github.com/devcontainers/features/pull/326

**Comment by justinyoo** [comment]
> @justinyoo Your postcreatCommad is reinstalling bicep [here](https://github.com/devkimchi/power-platfform-connector-via-apim/blob/main/.devcontainer/post-create.sh#L32). I wonder if that's causing the problem?

Yes, I do install bicep manually through `post-create.sh` because Azure CLI feature installs bicep differently.

> 
> ℹ️ We have bumped the major version for `common-utils` to `2` with support for Debian/Ubuntu, RHEL/Fedora/Rocky/CentOS, and Alpine Linux! Alsim. Alos, we have renamed `uid` & `gid` option to fix a bug. You may want to switch to that?

Oh, let me try that!

**Comment by samruddhikhandale** [comment]
Identified the bug, looks like `_REMOTE_USER_HOME ` in [here](https://github.com/devcontainers/features/blob/main/src/azure-cli/install.sh#L220) is not always correct.

These are the values when the `azure-cli` Feature is executed in your [dev config](https://github.com/devkimchi/power-platfform-connector-via-apim/blob/main/.devcontainer/devcontainer.json)👇 
<img width="362" alt="image" src="https://user-images.githubusercontent.com/24955913/208741591-4e5f9b24-d276-4392-bf73-c3ef4be8f729.png">

The value `_REMOTE_USER_HOME` should have been `/home/vscode`

Opened a bug in

**Comment by samruddhikhandale** [comment]
> Yes, I do install bicep manually through post-create.sh because Azure CLI feature installs bicep differently.

Created a PR with a temporary alternate solution to unblock you - https://github.com/devkimchi/power-platfform-connector-via-apim/pull/1

**Comment by samruddhikhandale** [comment]
Closing in favor of https://github.com/devcontainers/cli/issues/331
Fixing the CLI bug would automatically fix this.

**Comment by justinyoo** [comment]
Hmmm. I don't see the fix yet. I still have to manually install bicep through the command `az bicep install`.

**Comment by samruddhikhandale** [comment]
I was thinking of using https://github.com/devcontainers/cli/issues/331 to track this bug. However, lets have this open as well.
